sample_pages/esp/eids-esp.1.0.js

```diff
@@ -0,0 +1,39 @@
+function fetchAsyncSignals(mode, customFunction, customKey) {
+    console.log("Going to fetch signals");
+    var eids = "";
+    switch (mode) {
+        case 1:
+            if (pbjs && pbjs.getUserIdsAsEids && typeof pbjs.getUserIdsAsEids === "function") {
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: ['pbjs' is not defined.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15858)